### PR TITLE
Modified code to make the table more readable in report.md.

### PR DESCRIPTION
**Pull request by mark-velez**

### CODE REVIEW

File: reporter.py [hunk 1]
```diff
@@ -185,17 +188,37 @@ def export_log():
...
+
+            results[hpo_id][row_dict['table_name']]['log_id'] = str(row_dict['log_id'])  # for json serialize
+
+        for hpo_id in results.keys():
+            for table_name in results[hpo_id].keys():
+                all_log_items.append({'log_id':results[hpo_id][table_name]['log_id'], \
+                                      'hpo_id':hpo_id, 'table_name':table_name, \
+                                      'file_name':results[hpo_id][table_name]['filename'], \
+                                      'received':results[hpo_id][table_name]['received'], \
+                                      'parsing':results[hpo_id][table_name]['parsing'], \
```
Comment (mark-velez) [inline reporter.py]: At this line I hit the following error when it gets to a file not received log (i.e. `results[hpo_id][table_name]['received']==False`).

        'parsing':results[hpo_id][table_name]['parsing'], \
    KeyError: 'parsing'

**Comment by cukarthik** [comment]
@mark-velez can you try now? I pushed an update.

**Comment by mark-velez** [comment]
The value for `parsing` property in the log is sometimes set to numeric value `0` instead of a boolean value. One side effect is the column `Parsed File` may be checked when the file hasn't even been received.

image

**Comment by cukarthik** [comment]
Made the fix. I changed it so it put false instead of 0.

**Comment by mark-velez** [comment]
I'm now seeing many duplicate records. Sorry, this bug may have been there already, but I'm only seeing it now. Duplicates are not in the database, they are being written to the file by `export_log()`. My guess is that `results` is preserving the records from each iteration of [the outer loop](https://github.com/cumc-dbmi/pmi_sprint_reporter/blob/format_report/reporter.py#L196), causing them to get rewritten [here](https://github.com/cumc-dbmi/pmi_sprint_reporter/blob/format_report/reporter.py#L218).

image

**Pull request merged**